src/oas2/transformers/__tests__/params.test.ts

```diff
@@ -162,20 +162,21 @@ describe('params.translator', () => {
               },
               maxItems: 10,
               minItems: 1,
+              minLength: 1,
               type: 'array',
             },
             int: {
               description: 'desc',
               maximum: 3,
               minimum: 0,
+              minLength: 1,
```

Comment:
Maybe I'm not understanding this PR, but `minLength` is a string validation not an integer/number validation.

Reply:
Yeah, this test is likely wrong. I'll look into.